### PR TITLE
Enable native transfer in chainbridge erc20 pallet

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: frame/erc20/src/lib.rs [hunk 1]
```diff
@@ -81,7 +81,7 @@ decl_module! {
         }
 
         /// Transfers some amount of the native token to some recipient on a (whitelisted) destination chain.
-        /*  #[weight = 195_000_000]
+        #[weight = 195_000_000]
```
Comment (AndreiNavoichyk) [inline frame/erc20/src/lib.rs:84]: Remove commented code.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.8.0]
 ### Changed
 - Enabled inflation
+- Enabled native transfer on erc20 frame
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Move to vNext section.

File: frame/erc20/src/lib.rs (hunk 1 above)
Comment (karthikeyan-cere) [reply frame/erc20/src/lib.rs:84]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md]: Moved

**Review by AndreiNavoichyk — APPROVED** [review]

**Review by naure — APPROVED** [review]

**Pull request merged**